src/GeneralStateTestsFiller/stSubroutine/simpleSubroutineFiller.json

```diff
@@ -19,16 +19,18 @@
         "result" : {
           "1000000000000000000000000000000000000000" : {
             "balance" : "1000000000000000000",
-            "code" : "0x60045e005c5d00"
+            "code" : "0x60045e005c60016000555d00",
+            "storage": {
+            }
           }
         }
       }
     ],
     "pre" : {
       "1000000000000000000000000000000000000000" : {
         "balance" : "1000000000000000000",
-        "//code" : "This should jump into a subroutine, back out and stop.",
-        "code" : "(asm 0x04 JUMPSUB STOP BEGINSUB RETURNSUB)",
+        "//code" : "This should jump into a subroutine, store 0x01 at 0x00, back out and stop.",
+        "code" : "(asm 0x04 JUMPSUB STOP BEGINSUB 0x01 0x00 SSTORE RETURNSUB)",
```

Comment:
if there wouls be no subroutine opcodes I think those codes will be used for something else. 
we can't really use beginsub returnsub anymore ?

Reply:
I can replace with `:raw` code?

Reply:
I went with this approach in b38f7bd14f. Easy to remove if you prefer a different approach.

Reply:
Updated in fdd4da0f248e9e8cbe1ecb367718ad47b05b0ef6.